src/Cornelis/Utils.hs

```diff
@@ -24,10 +24,6 @@ import           Neovim hiding (err)
 import           Neovim.API.Text
 import           Neovim.Context.Internal (Neovim(..), retypeConfig)
 
-
-deriving via (Ap (Neovim e) a) instance Semigroup a => Semigroup (Neovim e a)
```

Comment:
Bad merge?

Reply:
Those instances have been upstreamed d in neovim-hs, afaict, I got duplicate instances after nixpkgs update, maybe that's the wrong fix though, maybe there should be a constraint instead. Sorry I'll fix that.

Reply:
You're right, sorry! I forgot! Maybe we should update the `package.yaml` file to account for the new version of `neovim-hs.`